admin/scss/bootstrap/_variables.scss

```diff
@@ -135,7 +137,7 @@ $body-color:                 $gray-dark;
 // Set the number of columns and specify the width of the gutters.
 
 // $grid-columns:               12;
-// $grid-gutter-width:          1.875rem; // 30px
+$grid-gutter-width:          2.4615rem; // now 32px, was 1.875rem; ~ 30px
```

Comment:
If we are to do that then we might as well just say its 32px. I guess I was trying to keep the rem thing going as much as possible so it can be changed globally based on the font sizing. I could change both $spacer and $grid-gutter-width to pixels values for now and see if that runs into any complications down the road like if the font-size changes, but it might be safer to leave as is with better comments.